firebase-project/src/App.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div id="app">
+    <nav-bar />
+    <router-view />
+  </div>
+</template>
+
+<script>
+import NavBar from "./components/navbar.vue";
+
+export default {
+  name: "App",
+  components: {
+    NavBar,
+  },
+};
+</script>
+
+<style>
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+}
+
+nav a {
+  font-weight: bold;
```

Comment:
Font weight can be manage by tailwind @mukeshsin

Reply:
only font-weight i convert all css into talwind css
in this component
@meanstack9716 
sir???